genesis-programs/src/lib.rs

```diff
@@ -7,10 +7,12 @@ extern crate solana_exchange_program;
 #[macro_use]
 extern crate solana_vest_program;
 
```

Comment:
Can you rebase on https://github.com/solana-labs/solana/pull/12490 (review for that PR would be great too!).  `genesis-programs/...` is no more!

Reply:
Done